std_msgs/Header header
std_msgs/Float64 closest_point
std_msgs/Float64 farthest_point
